core/domain/stats_jobs_one_off.py

```diff
@@ -389,7 +402,7 @@ def reduce(exp_id, stringified_values):
                                     ', '.join(map(str, state_names)),
                                     ', '.join(
                                         map(str, state_stats_mapping.keys()))))
-                            raise
+                            continue
```

Comment:
Please explain the reasoning behind the "continue" statement. I suppose this just ends up undercounting first-hit counts?

Reply:
@seanlip So, this try catch is basically handling the issue with the Pseudo-End states. Our way of computing the number of actual starts is computing the difference between the number of exploration starts and the max number of first hit counts among the states leading out of the initial state.
We were facing an error with a select few explorations which have a pseudo end state and also, the first state leads to this pseudo end state (which suggests that the exploration might be very tiny, or it was a test exploration made by people trying out the site).  What the continue statement effectively does here is: it excludes the pseudo end state from the states leading out of the first state and computes num actual starts using the rest of the states leading out of the initial state. Since we are also logging a lot of stuff, we can actually look up the explorations with this pseudo end state to figure out whats going on.

Reply:
OK, sounds good to me. I think it's worth double-checking that all dest_state names here are 'END', to ensure that the hypothesis is correct.